### ISSUE TITLE
Usability: Add old footer back to static pages for better navigation

### ISSUE THREAD

**Issue by fancyham** [hackforla/food-oasis#1520]
### Overview
After some discussion with the design team, we had some questions about 
- how many people knew how to use the ‘hamburger’ menu (which gives access to the static pages), and with
- the feeling of ‘dead end’ content on our static pages now that we lack a footer with navigation options

Currently our static pages, which are quite long (in particular, when viewed on mobile), end without a footer and the user has no where to continue if their question has not been answered on the current page and feels like a dead-end.
- #1288

We removed the footer back in Aug 2022:
- #1289 
- Dev implementation issue: #1302
and as part of
- #730 
but we may have been overzealous and removed too much.

### Request

- [x] Review if this is a usability problem that we should address
- [ ] Make updates, as needed

### Discussion and possible Solution(s)

Thinking that we’ll keep hamburger menu and duplicate the links in the footer?

- [ ] If confirmed that it’s a problem that can be solved adding the footer back:
  - [ ] Is old footer still good or does it need updating?
  - [ ] How does it look and feel on desktop and mobile?
  - [ ] Choose pages to add the footer back to.  (Perhaps all public pages except map and log-in?)

### Resources/Instructions
Take a look at archive.org for how it used to look and feel:
https://web.archive.org/web/20220603174439/https://www.foodoasis.la/faqs
![image](https://user-images.githubusercontent.com/3376957/235373243-94d7df75-45ec-44bb-a661-20242cefe0d2.jpeg)

### Action items
- [x] Discuss
- [x] Talk to usability folks (research team) or confirm that there’s a usability problem by concensus
- [ ] Decide what changes to make : Replace footer but update with current page links
  - [ ] What updates are needed for MVP
  - [ ] Nice-to-have: Footer seems to be in stacked-mode on wide screens… Review and update when it goes to flat-mode if necessary.
- [ ] Write up request for dev team to make updates (might be reverting #1302)

**Comment by staceyrebekahscott** [comment]
Discussed at Design meeting on 01/19. Waiting on presentation at Leads/PM meeting from Content team, as options for what the Footer looks like may change based on what Content comes up with.

**Comment by fancyham** [comment]
Based on the presentation, I think our footer stays relatively the same.  No major changes.

I suggest this is a usability problem and that we change the footer back to the way it was pretty easily in the meantime (it’s rolling back a change).

Then when we have new static page content and pages, we update the footer accordingly.

Not sure how to proceed — we removed the footer because it looked nicer without it, but it caused a usability problem with the static pages.

Are we comfortable just turning the footer back like it was, but leave it off the log-in screen?

Who do we need to consult to decide?

**Comment by staceyrebekahscott** [comment]
@fancyham Is it possible to add a screenshot of what the footer used to look like to this issue, just so some of the newer team members know what it will look like before we decide. 

And to be clear, we are not talking about the old footer from the archived site, but the most recent one where the options were horizontally along the bottom? (This is where the image would help).

**Comment by fancyham** [comment]
Here’s the old footer — let’s bring it back for the previous static pages

<img width="1366" alt="image" src="https://user-images.githubusercontent.com/3376957/226017037-4f9bd835-5e79-4831-a3fb-93588851758b.png">

**Comment by jeristella** [comment]
Work in progress. Here's a [document](https://docs.google.com/document/d/10wuhXM_jNETdLvZ7PZ6ArWes0gP6dtcohCFR1asAKxw/edit?usp=sharing) I created for specifications to send to the dev team for review!

**Comment by fancyham** [comment]
Cool -  let me know when you’re close to complete and would like a review together. 👍 I suggest including a quick list of the pages (and URLs) that you want to add the footer to.

**Comment by jeristella** [comment]
on it!

**Comment by fancyham** [comment]
Great!  Actually, please do put the contents of the document into a comment here — I just realized that doing so will make it easier to search for later using GitHub, if needed.  (Linked documents aren’t scanned by GitHub’s search engine)

Looks like the devs finished their work — good work, team!  Please check out what they built and do a QA (quality assurance) pass on it.  I noticed something unusual about the stacked version (I’ll tell you if you don’t catch it ;*) — hint: it’s the order of something) — so mention that on their dev issue in a comment and if you can, re-open the issue so that it gets taken care of (if you can’t re-open it, shoot a message to Hannah (dev) to let her know.

**Comment by jeristella** [comment]
FOLA Footer Redesign


Purpose:
Improves navigation for the users and not a deadend in the case that they still need more information / have questions
Exposes menu at the bottom to provide access to static pages 

Speak with Hannah from the dev team about these implementations:
Bring back footer in FOLA website 
Keep logo
Do not place on map page and log in page
Implement appropriate spacing for different screen sizes
Spacing between logo and the page names 
Horizontal footer on tablets 
Ensure that footer leads to the their respective static pages
Add footer in the following pages:
[About Us](https://www.foodoasis.la/about)
[Donate](https://www.foodoasis.la/donate)
[FAQs](https://www.foodoasis.la/faqs)
[Suggest New Listing](https://www.foodoasis.la/suggestion)
Dev notes 
Adjusted spacing to 4 em
Horizontal footer starts at 960px 
Horizontal footer stops expanding == 1200px

**Comment by fancyham** [comment]
Done!
